### DISCUSSION TITLE
Integrate with codegen

### DISCUSSION THREAD

**Discussion by Warxcell** [dotansimha/envelop#1750] [Q&A] [answered]
Specifically: how to gen typeof Context for usage in codegen?

**Comment by n1ru4l** [comment, marked as answer]
Hey @Warxcell, you can set a custom context for the typescript-resolvers plugin using the `contextType` configuration option. https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-resolvers#contexttype
